.github/workflows/tests.yml

```diff
@@ -0,0 +1,24 @@
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+name: tests
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x]
```

Comment:
not sure whether we need both versions, but it's a nice CI play so let's just keep it :)

Reply:
It is mostly to test the which versions are supported ;-) This fails if somebody uses a 1.14.x feature. Due to this line we are implicitly commited to support the two latest minor go versions ;)